funnel/views/sms_events.py

```diff
@@ -0,0 +1,125 @@
+from flask import request
+
+from twilio.request_validator import RequestValidator
+
+from baseframe import statsd
+from coaster.views import render_with
+
+from .. import app
+from ..models import SMS_STATUS, SMSMessage, db
+from ..transports.sms import validate_exotel_token
+
+
+@app.route('/api/1/sms/twilio_event', methods=['POST'])
+@render_with(template=None, json=True)
+def process_twilio_event():
+    """Process SMS callback event from Twilio."""
+
+    # Register the fact that we got a Twilio SMS event.
+    # If there are too many rejects, then most likely a hack attempt.
+    statsd.incr('phone_number.sms.twilio_event.received')
```

Comment:
We don't need to record these events, because all endpoints are logged by default (in Baseframe).

Reply:
We did the same in Email also. The idea is to use Kibana's dashboard and group these events together than trying to cobble it from various other sources for correlating them.